src/Countries/US/Georgia/GeorgiaIncome/V20190101/GeorgiaIncome.php

```diff
@@ -113,18 +113,21 @@ private function getStandardDeduction()
 
     private function getPersonalAllowance()
     {
-        if ($this->tax_information->personal_allowances === 0 || !array_key_exists($this->tax_information->filing_status, self::PERSONAL_EXEMPTION_ALLOWANCES)) {
+        $personal_allowances = $this->tax_information->personal_allowances;
```

Comment:
We tend to avoid using temporary variables just to increase readability but I personally am fine with this.